### PR TITLE
Add wings42 rev2

### PR DESCRIPTION
**Pull request by yfuku**
## Description
Add wings42 rev2

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wings42/rev2/keymaps/via/config.h [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#pragma once
```
Comment (drashna) [inline keyboards/wings42/rev2/keymaps/via/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/wings42/rev2/keymaps/via/config.h (hunk 1 above)
Comment (yfuku) [reply keyboards/wings42/rev2/keymaps/via/config.h:16]: @drashna 
Thanks for reviewing it.
I've added the license header.

**Review by drashna — APPROVED** [review]

**Comment by yfuku** [comment]
It would be helpful if you could give us a review.

**Comment by yfuku** [comment]
I'm sorry. I closed the PR by mistake.
Please continue your review.

**Comment by yfuku** [comment]
If you have time, please review.
Thank you very much.

**Review by tzarc — APPROVED** [review]

**Pull request merged**